Makefile

```diff
@@ -801,7 +801,7 @@ test-code-android:
 
 # Runs checkstyle and lint on the java code
 .PHONY: android-check
-android-check : android-checkstyle run-android-nitpick android-lint-sdk android-lint-test-app android-ktlint
+android-check : android-ktlint android-checkstyle android-lint-sdk android-lint-test-app run-android-nitpick
```

Comment:
`android-ktlint` should also be dependent on the `platform/android/gradle/configuration.gradle`